src/components/PrimaryNav/PrimaryNavItem/PrimaryNavItem.spec.tsx

```diff
@@ -3,6 +3,7 @@ import {render, screen} from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
 import {PrimaryNavItem} from './index';
 import {PrimaryNav} from '../PrimaryNav';
+import {Badge} from '~/main';
```

Comment:
As `main.ts` is only there for compatibility reasons, I prefer to use

```suggestion
import {Badge} from '~/components';
```